tasks/main.yml

```diff
@@ -30,3 +30,9 @@
     state: started
     enabled: true
   loop: "{{ __keylime_server_services }}"
+  loop_control:
+    # The services must be started in a specific order, and they may
+    # depend on something the previous one will do, such as creating
+    # a CA and/or some certificates, in some configurations, so let
+    # us add a delay between them.
+    pause: 5
```

Comment:
getting timing with `sleep` and the like is always tricky - is there some file/service that the role can look for?

Reply:
Yeah, I *just* updated it to check for the verifier being accessible via its port, instead of this, which would be more reliable.